CRM/Admin/Form/Setting.php

```diff
@@ -95,6 +95,8 @@ public function buildQuickForm() {
 
     $descriptions = array();
     $settingMetaData = $this->getSettingsMetaData();
+    //Load readonly for the input fields whose values are overridden in civicrm_settings.php.
+    $overriddenSettings = CRM_Utils_Array::value('civicrm_setting', $GLOBALS);
```

Comment:
i would remove this and in 110 use something like `Civi::settings()->getMandatory($setting);` that should tell you if its overrriden i think iirc

Reply:
It would be like this https://github.com/civicrm/civicrm-core/blob/master/CRM/Admin/Form/Setting/Debugging.php#L57

Reply:
Agreed and updated. Thanks 👍